configure

```diff
@@ -1220,6 +1219,12 @@ def configure_intl(o):
         pprint.pformat(icu_config, indent=2) + '\n')
   return  # end of configure_intl
 
+def configure_inspector(o):
+  disable_inspector = (options.without_inspector
+                       or options.with_intl in (None, 'none')
+                       or options.without_ssl)
```

Comment:
I realize this is nitpicking but since we put operators at the end of the line everywhere else can you do it here too?